### ISSUE TITLE
Feature request: Show output produced by `git commit` interactively

### ISSUE THREAD

**Issue by ulope** [gitx/gitx#140]
First: Awesome that you've taken up working on gitx, thanks a lot!

For those of us using tools like [`pre-commit`](https://pre-commit.com) it would be useful to have an interactive view of the output being generated during the call to `git commit` (since it can, depending on the configuration, take quite some time for that to return).

**Comment by tiennou** [comment]
Yep, that kind of thing was what sparked the recent "task rework". It's not quite there yet, but maybe one day 😉.
